### PR TITLE
Implementation of BinaryCrossEntropy/CrossEntropy/InceptionLoss using label smoothing.

### PR DESCRIPTION
**Pull request by dbogunowicz**
CrossEntropy losses (Categorical) now can take `label smoothing` are as parameter.

### CODE REVIEW

**Comment by dbogunowicz** [comment]
Just tested the current commit using command:

`python integrations/pytorch/train.py \
    --recipe-path ~/pruning_resnet50.yaml \
    --arch-key resnet50 --dataset imagenette \
    --train-batch-size 128 --test-batch-size 512`
  
  (pruning resnet50 on imagenette)
  
  ...and just for fun checked whether the loss graph looks aight:

image

**Comment by dbogunowicz** [comment]
Will not be needed if we move repo to be compatible with pytorch 1.10.x (where label smoothing is explicitly implemented).
